pandas_gbq/gbq.py

```diff
@@ -833,6 +839,7 @@ def read_gbq(
     max_results=None,
     verbose=None,
     private_key=None,
+    progress_bar=None,
```

Comment:
Let's call the argument `progress_bar_type` for consistency with google-cloud-bigquery.

Also, I believe it's safe to set the default to `tqdm` even if the library is not installed.

Reply:
👍Good points, done and done.